src/superphot_plus/model/classifier.py

```diff
@@ -407,26 +280,22 @@ def evaluate_epoch(self, iterator):
 
         return epoch_loss / len(iterator), epoch_acc / len(iterator)
 
-    def test(self, test_features, test_classes, test_names, test_group_idxs, save_path):
+    def test(self, save_path):
```

Comment:
As mentioned above, I would suggest having this take in the test data set explicitly.

You could rename `evaluate()` and use this any time you want to evaluate a set of data (test or otherwise).